### ISSUE TITLE
Loss inf

### ISSUE THREAD

**Issue by David-19940718** [meituan/YOLOv6#526]
### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar bug report.

### YOLOv6 Component

Training（训练）

### Bug

The cls_loss got inf while the batch-size is setted lower, e.g. 1 <= bs <= 4, which reason lead to this?
Can u give advice to me how to set these hyper-parameters?

### Environment

Using the latest version util now release.

### Minimal Reproducible Example

None

### Additional

By the way, the same dataset training in yolov5 project is fine.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by Chilicyy** [comment]
@David-19940718 Which config config did you use?

**Comment by David-19940718** [comment]
> @David-19940718 Which config config did you use?

yolov6m-finetune.py

**Comment by Chilicyy** [comment]
You can add `--write_trainbatch_tb` in the training command for visualization.
